green/__main__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
+
+from .cmdline import main
+import sys
```

Comment:
'import sys' should be above 'from .cmdline import main', with a blank line between them